java/runtime/src/main/java/org/ray/runtime/runner/worker/DefaultDriver.java

```diff
@@ -1,26 +1,42 @@
 package org.ray.runtime.runner.worker;
 
-import org.ray.api.Ray;
-
 /**
  * The main function of DefaultDriver.
  */
 public class DefaultDriver {
 
-  //
-  // " --node-ip-address=" + ip
-  // + " --redis-address=" + redisAddress
-  // + " --driver-class" + className
-  //
+  /**
+   * Command line:
+   *
+   * <p>java -ea org.ray.runtime.runner.worker driverClassName [driverArg1,driverArgs2,...]
```

Comment:
1) no need to add `-ea`
2) `[driverArg1,driverArgs2,...]`: maybe it's easier to just separate them by space